brambling/views/core.py

```diff
@@ -47,7 +47,8 @@ def get_context_data(self):
             re_dict = dict((e.pk, e) for e in registered_events)
             orders = Order.objects.filter(
                 person=user,
-                bought_items__status=BoughtItem.RESERVED,
+                bought_items__status__in=(BoughtItem.BOUGHT,
+                                          BoughtItem.RESERVED),
```

Comment:
Not really a query about this PR, but I don't know what this change actually does without understanding that.